lib/src/components/NxNavigationDropdown/NxNavigationDropdown.scss

```diff
@@ -30,6 +30,7 @@
 
   .nx-dropdown-menu {
     border: var(--nx-border-width) solid var(--nx-color-global-sidebar-2-border);
+    max-height: unset;
```

Comment:
RSC-1800. This caused the nav dropdowns in IQ new-header prototypes not to work correctly, though this was also fixable on the IQ side.